Maskbook/Arweave/ArweaveUploader.swift

```diff
@@ -0,0 +1,124 @@
+//
+//  ArweaveUploader.swift
+//  Maskbook
+//
+//  Created by xiaojian sun on 2022/6/16.
+//  Copyright © 2022 dimension. All rights reserved.
+//
+
+import Foundation
+import CryptoSwift
+
+class ArweaveUploader {
```

Comment:
Why we need this class?

Reply:
In `ArweaveUploadHandler`  I faced some restrictions.
I have to pass muti parameter in some functions, such as option , item , tx.
And It's a little diffcult to implement progress.
In `ArweaveUploader` I have full freedom to do things.

Reply:
The protocol `FileServiceUploadHandler` function without mutating give these restrictions.
After I finish `ArweaveUploader `, I may move the code to `ArweaveUploadHandler `

Reply:
As we are in the context of `Task.detached { }` now, you can have all you need, `mutating` is not necessary.